docs/guide/areas-and-widgets/core-widgets.md

```diff
@@ -1,128 +1,263 @@
+by 1140px, for the `src` value.
+
+You can change this in the area field widget options, using another image size name.
+
+```js
+// modules/@apostrophecms/home-page/index.js
+// In area field configuration options
+widgets: {
+  '@apostrophecms/image': {
+    size: 'one-half'
+  }
+}
+```
+
+The sizes available by default are:
+
+| Name | Maximum width | Maximum height |
+| ------ | ------ | ------ |
+| `max` | 1600px | 1600px |
+| `full` | 1140px | 1140px |
+| `two-thirds` | 760px | 760px |
+| `one-half` | 570px | 700px |
+| `one-third` | 380px | 700px |
+| `one-sixth` | 190px | 350px |
+
+The final image size name is `original`, which delivers the original image file. This should be used carefully since it could be very large and slow to download.
 
-For fine-grained control in legacy browsers, `size` option can still be set to `one-sixth`, `one-third`, `one-half`, `two-thirds`, `full` or `max` (1600 pixels wide) when adding the widget to an area.
+## Video widget
 
-## Configuring the video widget
+The core video widget accepts a video URL and fetches the embed code to display it. Most major video hosts are supported by default. The widget uses the oEmbed data format, so you can find a full list of supported services [on the oEmbed website](https://oembed.com/#section7).
 
-While the video widget looks better out of the box, you can configure a `className` option for that as well if you wish.
+## HTML widget
 
-## Configuring the HTML widget
+**...or, how to get access to the editing interface when embedded HTML breaks it.**
+
+The HTML widget allows content editors to embed raw HTML directly into a template. This can be helpful if they may need to add third-party features (e.g., sign-up forms). This can also be very dangerous since there are no limits to what they can add. Bad Javascript in embedded HTML can break the user interface, making it impossible to remove the bad code.
+
+**Think carefully before providing this widget option to editors and make them aware of the risks.**
+
+There is a safety mechanism in case things do go wrong. If embedded HTML breaks the Apostrophe interface, append the query parameter `safemode=1` to the end of the URL. The HTML widget will not render its contents and editors will be able to edit it to remove or fix the embed.
+
+```
+https://example.net/broken-page?safemode=1
+```
 
-There's nothing to configure! But, note that if you paste a bad embed code that breaks the tag balancing of the markup or otherwise damages the page, you will need a way to get control back.
 
 To do that, access the page with `?safemode=1` at the end of the URL. Then you will be able to edit the widget and remove the offending content.
+
+## Setting a CSS class on core widgets
+
+It is a front end development best practice to have a consistent `class` naming system, regardless of what that system is. There are two options to set classes on core widgets to support this practice and avoid overwriting the whole template to do so.
```

Comment:
```suggestion
There are two options to set classes on core widgets.
```
Idk I just think we could cut to the chase here. Leave best practice out with showing rather than telling.